src/server/snapshot.cc

```diff
@@ -244,7 +244,7 @@ void SliceSnapshot::PushFileToChannel(DbIndex db_index, bool should_compress,
       zstd_serializer_.reset(new ZstdCompressSerializer());
     }
 
-    if (auto comp = zstd_serializer_->Compress(payload); comp) {
+    if (auto comp = zstd_serializer_->Compress(payload)) {
```

Comment:
I made this on purpose, see https://github.com/dragonflydb/dragonfly/pull/508

Not sure what side you will join 🙂 

In this case, with declaring a new variable, it probably doesn't matter

Reply:
Ok I will undo this change